src/backend/src/modules/pia-intake/dto/pia-intake-find-query.dto.ts

```diff
@@ -25,6 +27,37 @@ export class PiaIntakeFindQuery {
   @Transform(({ value }) => value.trim())
   readonly searchText?: string = '';
 
+  @ApiProperty({
+    required: false,
+    type: PiaIntakeStatusEnum,
+    example: piaIntakeEntityMock.status,
+  })
+  @IsString()
+  @IsOptional()
+  @Length(0, 20)
+  @Transform(({ value }) => value.trim())
+  readonly filterByStatus?: string = '';
+
+  @ApiProperty({
+    required: false,
+    type: GovMinistriesEnum,
+    example: piaIntakeEntityMock.ministry,
+  })
+  @IsString()
+  @IsOptional()
+  @Length(0, 100)
+  readonly filterByMinistry?: string = '';
+
+  @ApiProperty({
+    required: false,
+    type: String,
+    example: 'onlyMyPia',
+  })
+  @IsString()
+  @IsOptional()
+  @Length(0, 20)
+  readonly filterByDrafter?: string = '';
```

Comment:
filterByDrafter isn't a right naming terminology. Also, what is the data that it expects? Please create an enum of it, allowing any random string.

Reply:
add a new enum PiaFilterDrafterByCurrentUserEnum, change to filterPiaDrafterByCurrentUser